src/model/operation/transform.js

```diff
@@ -2184,7 +2804,16 @@ function makeMoveOperationsFromRanges( ranges, targetPosition ) {
 	return operations;
 }
 
-function makeMoveOperation( range, targetPosition ) {
+/**
+ * Helper function. Creates a move operation for given `range` to move and `targetPosition`.
+ *
+ * @private
+ * @param {module:engine/model/range~Range} range
+ * @param {module:engine/model/position~Position} targetPosition
+ * @returns {module:engine/model/operation/moveoperation~MoveOperation}
+ * @ignore
+ */
+function _makeMoveOperation( range, targetPosition ) {
```

Comment:
Maybe move constructor should look like this?

Reply:
Yeah, well, maybe but probably nope. Because then you'd be able to pass non-flat range and you'd have to check it. But OTOH it would simplify places where transformation follows this scheme:

1. Change `sourcePosition` + `howMany` into range.
2. Transform that range.
3. Read new `sourcePosition` and `howMany` from the transformed range and change them in the operation.

Reply:
Okay I removed this function completely.